MyMethods.cs

```diff
@@ -0,0 +1,257 @@
+﻿using System;

```

Comment:
Przebudujmy caly ten plik na osobne podpliki - bedzie sie latwiej sprawdzalo to na pewno ;)

Wklejam jak ulozylbym foldery/klasy:

```
App
	Loggers
		LoggerInterface
		Logger implements LoggerInterface
		FileLogger extends
	Readers
		ReaderInterface
		Reader implements ReaderInterface
	Handlers
		HandlerInterface
		Handler implements HandlerInterface
		ExcelHandler extends Handler
		ChromeHandler extends Handler
	Extractors
		ExtractorInterface
		Extractorimplements ExtractorInterface
		GoogleExtractor extends Extractor
	Exceptions
		ExceptionHandler	
```